src/api/airtableApi.jsx

```diff
@@ -42,8 +42,8 @@ export const listRecords = async options => {
     // sort by title
     params = {
       ...params,
-      'sort[0][field]': 'title',
-      'sort[0][direction]': 'desc',
+      'sort[0][field]': 'index',
```

Comment:
We rely on airtable query params to sort the blog titles by index name.

I added the index name to the airtable table as an integer:
![image](https://user-images.githubusercontent.com/6400323/234258927-ea5255f3-b728-4315-92ac-ed8d78deb79c.png)